dts/bindings/display/hub12.yaml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2025 Siratul Islam <sirat4757@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  HUB12 interface monochrome LED matrix display driver.
+
+  The HUB12 interface uses SPI for data transfer to shift registers,
+  with additional GPIO pins for row address selection (PA, PB),
+  output enable (PE), and data latching (PLAT).
+
+compatible: "hub12"
```

Comment:
Has to be vendor, device. Also use the specific chip, that is used, if there are multiple chips, use the one that this is based (the original not the clones) on.

Reply:
is there at least some documentation you could refer there then